app/assets/stylesheets/logixRelease.scss

```diff
@@ -271,6 +271,10 @@ a.btn.btn-outline-danger {
   padding-top: 40px;
 }
 
+.align-center{
```

Comment:
`Opening curly brace `{` should be preceded by one space `

Reply:
Done!